app/jobs/calendar_importer_job.rb

```diff
@@ -9,6 +9,11 @@ class CalendarImporterJob < ApplicationJob
     report_error exception, "Calendar URL is not accessible"
   end
 
+  rescue_from ActiveRecord::ActiveRecordError do |exception|
+    raise exception if Rails.env.development? && @silence_db_exceptions == false
```

Comment:
Assume you've thought about this but does this not want to activate for env == test too?